app/pages/about/acknowledgements.spec.js

```diff
@@ -0,0 +1,17 @@
+/* eslint prefer-arrow-callback: 0, func-names: 0, 'react/jsx-filename-extension': [1, { "extensions": [".js", ".jsx"] }] */
+
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import Acknowledgements from './acknowledgements';
+
+describe('Acknowledgements', function () {
+  it('renders without crashing', function () {
```

Comment:
similar to https://github.com/zooniverse/Panoptes-Front-End/pull/4224/files#diff-68a4f0f6c9db644e47eab002cca11ca1R9